cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/dao/EntityUtils.java

```diff
@@ -59,4 +60,16 @@ public static String validateEntities(Collection<? extends CradleEntity> entitie
 		return null;
 	}
 
+	public static <T extends CradleEntity> List<T> toCompleteEntitiesCollection(MappedAsyncPagingIterable<T> resultSet)
+			throws CradleStorageException
+	{
+		try
+		{
+			return DaoUtils.toList(resultSet);
+		}
+		catch (Exception e)
+		{
+			throw new CradleStorageException("Error while converting result set to collection", e);
```

Comment:
"Error while converting result set to collection" > "Error while converting result set to list"